### PR TITLE
CSS and jesting

### PR DESCRIPTION
**Pull request by azdoolittle**

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-console */
 const mongoose = require('mongoose');
 const channelSchema = require('./schema.js');
+require('dotenv').config();
```
Comment (priscilagonzalez) [inline db/index.js]: Did you add this line for later? Sorry I am not familiar with dotenv.

Comment (azdoolittle) [reply db/index.js]: Yes, this is the module that creates variables to protect sensitive data like API keys and passwords

[dotenv](https://www.npmjs.com/package/dotenv)

File: tests/MongoDB.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+describe('insert', () => {
+  let connection;
+  let db;
+
+  beforeAll(async () => {
+    connection = await mongoose.connect('mongodb://localhost/test', {
+      useUnifiedTopology: true, useNewUrlParser: true,
+    });
+    db = await mongoose.connection;
```
Comment (Kraig5) [inline tests/MongoDB.test.js]: Why two DB connections instead of just one?

Comment (azdoolittle) [reply tests/MongoDB.test.js]: @Kraig5 Good question. I just took this out and the code still works. This is what may have been interfering with my db connections and dotenv yesterday.

**Pull request merged**